### DISCUSSION TITLE
Unwanted Autoreplies by Bot in every Text-Channel

### DISCUSSION THREAD

**Discussion by W4nT4n** [jagrosh/MusicBot#1095] [Troubleshooting] [answered]
I am currently trying to setup the Jmusicbot via setup for my discord server. Everything is running smoothly (great bot, thank you for your work!) but more often than not the Bot will autoreply with a "hi to you too" even if the message it is reacting to didnt include a hi.

For example: 

Me: And this is why we need to find a way to...
Bot: Hi to you too!

I get that it is reacting to the "hi" in "this" but how do I make it stop or at least limit its replies to a text channel specifically reserved for music requests? Currently it is replying to public and private channels, so it keeps on popping up all over my server which is kinda messy. 

Thank you kindly for your time

**Comment by jagrosh** [comment, marked as answer]
The behavior you are describing is **not JMusicBot**. Either you are running another bot, or you have leaked the bot's token and somebody else is running a bot. I recommend **re-generating your bot's token immediately**, and **re-downloading the JMusicBot jar** from the releases page to make sure you aren't running something unintended.

**Reply by W4nT4n** [reply]
Ho boy, thank you for your quick reply!
